packages/lib-panoptes-js/src/resources/projects/projects.js

```diff
@@ -30,6 +27,23 @@ const projects = {
     return panoptes.get(`${projectsEndpoint}/${projectId}`, queryParams)
   },
 
+  getBySlug: (params) => {
+    const queryParams = params || {}
+    
+    if (queryParams.slug && typeof queryParams.slug !== 'string') return handleError('Projects: Get request slug must be a string.')
+    if (!queryParams.slug && (isBrowser() || process.env.NODE_ENV === 'test' && global.window)) {
```

Comment:
Since `||` short-circuits, the linter prefers:

```
(!queryParams.slug && (isBrowser() || (process.env.NODE_ENV === 'test' && global.window)))
```